release/action.yaml

```diff
@@ -36,6 +36,7 @@ runs:
     - name: Setup tools
       uses: open-turo/action-setup-tools@v1
     - name: Release
+      id: release
```

Comment:
This id is the value used in the `outputs/new-release-published/value` block (and other related ones)

Reply:
just for understanding, `steps.release.outputs.new_release_major_version` currently will not work because it cannot identify release step?

Reply:
I believe so.

https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#outputs-for-composite-actions
and https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsid
seem to suggest that you need to explicitly define the ID and that it doesnt exactly convert the `name` or anything

https://stackoverflow.com/questions/66762605/what-is-the-difference-between-name-and-id-in-github-actions